provider/datagen/src/tests/make_testdata.rs

```diff
@@ -2,10 +2,10 @@
 // called LICENSE at the top level of the ICU4X source tree
 // (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
+use crate::fs_exporter::serializers::Json;
```

Comment:
maybe do something like `use crate as icu_datagen` to make this look more external?

Reply:
Well it's not external any more and some APIs we use might not be ones that are publicly available, so probably better to not make the alias here.

Reply:
Internal APIs you can still access through `crate` if you want. Just these are all reexports, and I think it's weird accessing them through `crate`. If you don't want to access them through `icu_datagen`, maybe just inline, i.e. `icu_provider_fs::export` instead of `crate::fs_exporter`.

Reply:
Done